src/test/java/games/strategy/triplea/ui/TimespanDialogTest.java

```diff
@@ -0,0 +1,39 @@
+package games.strategy.triplea.ui;
+
+import static org.junit.Assert.assertNull;
+import static org.junit.Assert.assertTrue;
+
+import java.time.Instant;
+
+import org.junit.Test;
+
+import games.strategy.triplea.ui.TimespanDialog.TimeUnit;
+
+public class TimespanDialogTest {
+
+  @Test
+  public void testForeverReturnsNull() {
+    assertNull(TimeUnit.FOREVER.getInstant((int) (Math.random() * Integer.MAX_VALUE)));
```

Comment:
Random inputs tend to be a smell in unit tests.  I would just test the boundary conditions (`Integer#MIN_VALUE`, `Integer#MAX_VALUE`) and zero if you're trying to demonstrate that this method will always return `null`.